src/Utilities/ToolLocationHelper.cs

```diff
@@ -105,13 +105,18 @@ public enum TargetDotNetFrameworkVersion
         /// </summary>
         Version48 = 14,
 
+        /// <summary>
+        /// version 4.8.1
+        /// </summary>
+        Version481 = 15,
```

Comment:
Reference this around line 2052 to be the new Latest

Reply:
Ping on this, is it a bad idea or just missed?

Reply:
In theory the fallback rule would save this if the user didn't have 481 installed, so we should be safe to bump VersionLatest to the next version.